website/server/libs/analyticsService.js

```diff
@@ -183,6 +183,8 @@ function _sendDataToAmplitude (eventType, data) {
     logger.info('Amplitude Event', amplitudeData);
   }
 
+  if (loggerOnly) return new Promise();
```

Comment:
@SabreCat I think instead of returning a promise, we could:

1. Transform the function in an `async` one and then just `return`
2. Otherwise we can keep it a normal function but instead of `new Promise()` do `Promise.resolve(returnValue)`

Reply:
@SabreCat given the extra events being sent to amplitude with the a/b tests maybe we should finish this?

Reply:
I went with `Promise.resolve`, is there anything more meaningful than `null` we could return?

Reply:
I think null is fine if we don't have anything more meaningful to return